tests/helper_methods.py

```diff
@@ -0,0 +1,108 @@
+import os
+from pathlib import Path
+import shutil
+from typing import Tuple
+from packaging.version import parse as parse_version
+from ert.run_models.everest_run_model import EverestRunModel
+from everest.config import EverestConfig
+from ert.ensemble_evaluator import EvaluatorServerConfig
+from ert.shared import __version__ as ert_version
+from everest.api import EverestDataAPI
+import polars as pl
+import io
+
+
+def setup_environment(snapshot, source_root: Path, experiment_name: str, config_file: str) -> Tuple[EverestConfig, str]:
+    reference_path = source_root / snapshot.snapshot_dir
+    references = [f.stem for f in reference_path.glob("*.csv")]
+    release_name = parse_version(ert_version)
+    filtered_sorted_references = sorted(
+        ref for ref in references if parse_version(ref) <= release_name
+    )
+    if filtered_sorted_references:
+        reference = filtered_sorted_references[-1]
+    else:
+        raise ValueError(
+            f"There is no reference snapshot for or before release {release_name}"
+        )
+    reference_file = f"{reference}.csv"
+
+    os.chdir(os.path.join(source_root, "data", "drogon", experiment_name, "everest", "model"))
```

Comment:
source_root is already of type `Path` so it should be possible to use `pathlib` here.